ui/admin/tests/acceptance/accounts/change-password-test.js

```diff
@@ -77,6 +77,11 @@ module('Acceptance | accounts | change password', function (hooks) {
 
   test('can change password for account', async function (assert) {
     assert.expect(2);
+    // TODO: Failing due to a11y violation while in dark mode.
```

Comment:
Do we have a jira ticket to cover such investigation? if so It will good to add the jira ticket link, so no need to repeat all the 
```
// TODO: Failing due to a11y violation while in dark mode.
    // Investigating issue with styles not properly
    // being applied during test.
```

and we can say something like `// TODO: Issue fix ICU-12345`